### DISCUSSION TITLE
Do I still need services for classic clients?

### DISCUSSION THREAD

**Discussion by rulev** [uyuni-project/uyuni#6131] [Q&A] [answered]
Good day!

Do I still need some of those services running on my Uyuni server if I have only salt clients?
jabberd-c2s.service
jabberd-router.service
jabberd-s2s.service
jabberd-sm.service
jabberd.service
osa-dispatcher.service
spacewalk-wait-for-jabberd.service

AFAIR it is for classic (rhnsd) client support, am I right?

**Comment by mcalmer** [comment, marked as answer]
Yes, this is for the traditional client. You can disable them already and spacewalk-service will not start them again.

$> systemctl disable jabberd
$> systemctl stop jabberd
$> systemctl disable osa-dispatcher
$> systemctl stop osa-dispatcher

I think they will go away completely in the near future.
